db/migrate/20230803172129_add_devise_to_users.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :encrypted_password, null: false, default: ""
+
```

Comment:
- [OPTIONAL] Great job on your work! 👏👏 However, there seems to be an error occurring during the migration process ⚠️. It appears that you missed adding an email column here for `Database authenticatable`. I highly recommend to fix this for a smooth users experience and getting the expected results. Could you please try the following to fix this? 

```ruby
  ## Database authenticatable
    t.string :email,              null: false, default: ""
    t.string :encrypted_password, null: false, default: ""

```
![image](https://github.com/ridabensalem/blog-app/assets/106119105/c70e250d-58fd-4bdd-b11b-3519b9ce2ba8)